tests/test_top_level_editable.py

```diff
@@ -17,7 +17,7 @@ def get_dependencies(self, ireq):
 
 @pytest.fixture
 def mocked_repository():
-    return MockedPyPIRepository(["--index-url", PyPIRepository.DEFAULT_INDEX_URL])
+    return MockedPyPIRepository(["--index-url", ""])
```

Comment:
```suggestion
    return MockedPyPIRepository(["--no-index"])
```

I'd suggest `--no index` option.

Reply:
Sorry, replaced.